sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Last.scala

```diff
@@ -29,15 +29,18 @@ import org.apache.spark.sql.types._
  * a single partition, and we use a single reducer to do the aggregation.).
  */
 @ExpressionDescription(
-  usage = "_FUNC_(expr,isIgnoreNull) - Returns the last value of `child` for a group of rows.")
+  usage = """_FUNC_(expr,isIgnoreNull) - Returns the last value of `child` for a group of rows.
+    _FUNC_(expr,isIgnoreNull=false) - Returns the last value of `child` for a group of rows.
```

Comment:
While you are added could you also add a line about this being a non-deterministic function